recipe/meta.yaml

```diff
@@ -18,21 +18,21 @@ requirements:
   host:
     - pip
     - python
+    - jpype1
   run:
     - python
-
-test:
-  imports:
-    - pyreportjasper
-    - test
+    - jpype1
 
 about:
-  home: https://github.com/PyReportJasper/pyreportjasper
```

Comment:
Why change this to the docs page instead of the github page?

Reply:
As it is the official documentation page